test/test_support.ts

```diff
@@ -12,6 +12,7 @@ const OPTIONS: ts.CompilerOptions = {
   noResolve: true,
   skipDefaultLibCheck: true,
   noEmitOnError: true,
+  experimentalDecorators: true,
```

Comment:
another option to add here is `emitDecoratorMetadata`, to match our internal TS flags.